### PR TITLE
Moved installation from sources to contribute

### PR DESCRIPTION
**Pull request by lzap**
I am keeping the 3.5 section because deleting it would shift all other chapters and break links. Our menu is awful. :-)

### CODE REVIEW

File: _includes/manuals/nightly/3.4_install_from_source.md [hunk 1]
```diff
@@ -1,81 +1 @@
...
-
-{% highlight bash %}
-$ git clone https://github.com/theforeman/hammer-cli-foreman.git
-$ cd hammer-cli-foreman
-$ rake install
-$ cd ..
-{% endhighlight %}
-
-You can install other hammer plugins via any of the methods mentioned above.
+Installation from sources was moved to [contribute](/contribute.html) page.
```
Comment (ehelms) [inline _includes/manuals/nightly/3.4_install_from_source.md]: So we are saying this method is still supported, we just moved the docs?

Comment (lzap) [reply _includes/manuals/nightly/3.4_install_from_source.md]: Hmm I can probably state here that this is meant for development setups.

**Comment by lzap** [comment]
'bout this?

File: _includes/manuals/nightly/3.4_install_from_source.md (hunk 1 above)
Comment (ehelms) [reply _includes/manuals/nightly/3.4_install_from_source.md]: I'm good with that change and thus stating it here. @tbrisker ?

Comment (tbrisker) [reply _includes/manuals/nightly/3.4_install_from_source.md]: Makes sense to me to mention this isn't supported for production setups.

File: contribute.md [hunk 2]
```diff
@@ -12,6 +12,107 @@ The Foreman has a growing list of community events - check the [events
 page](/events) for more details. If there's nothing suitable in your area, why
 not consider [creating an event](https://github.com/theforeman/theforeman.org/blob/gh-pages/_data/events.yml)!
 
+## Installing from Source
```
Comment (tbrisker) [inline contribute.md]: This should probably be combined with the "Setting up development environment" section below, a lot of it is duplicated

Comment (lzap) [reply contribute.md]: Merged.

**Comment by lzap** [comment]
Any other comments?

**Comment by lzap** [comment]
We somehow dropped the ball for this one.

**Pull request closed without merging**